lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable self-paced courses.
+    'ENABLE_SELF_PACED_COURSES': False,
```

Comment:
I'd lean towards putting it into `openedx.core.djangoapps` (if it's needed by LMS and Studio), or directly into `lms.djangoapps` (if it's lms specific).

The problem with `common/djangoapps` is that in order to import from that directory, we need to muck with `sys.path`, whereas `openedx` is an actual python module. I want to get rid of the `sys.path` changes that we make, so I want to move as much as possible into `openedx`.